pymathics/graph/parametric.py

```diff
@@ -198,6 +198,9 @@ def eval(
 
 class CompleteGraph(_NetworkXBuiltin):
     """
+    <url>
+    :WMA:https://reference.wolfram.com/language/ref/CompleteGraph.html
```

Comment:
NetworkX links would be to `nx.complete_graph` and `nx.complete_multiparatie_graph`.